### DISCUSSION TITLE
自定义表头图标，图标和文字重合问题

### DISCUSSION THREAD

**Discussion by RJM1996** [antvis/S2#2622] [Q&A]
<img width="447" alt="image" src="https://github.com/antvis/S2/assets/33800707/27b9129b-806c-4872-8c0d-620bfc7d35f4">
自定义表头图标时，图标和文字的重合问题如何解决呢？

**Comment by RJM1996** [comment]
@lijinke666 求助大佬

**Comment by lijinke666** [comment]
请提供复现链接, 谢谢, 光看一张图片如何回答你?
